core/src/main/java/com/kin/ecosystem/core/data/blockchain/BlockchainSourceImpl.java

```diff
@@ -311,6 +313,11 @@ public void onResponse(Balance response) {
 			});
 	}
 
+	private Payment toPaymentObj(final String orderID, PaymentInfo data) {
```

Comment:
toPaymentObj should be in Payment.java - used twice every future update will need to be updated in several places

Reply:
it used only once.